lib/dialects/mysql/query-generator.js

```diff
@@ -353,7 +353,11 @@ var QueryGenerator = {
   },
 
   geometrySelect: function(column) {
-    return 'AsText(' + this.quoteIdentifiers(column) + ') AS ' + column;
+    return {
+      as: column,
+      fn: 'AsText',
+      cols: [column]
+    };
```

Comment:
This looks like completely new syntax?
Would `return [sequelize.fn('AsText', sequelize.col(column)), column]` not accomplish the same thing?